### PR TITLE
(SYS-4599) Added Prettier

### PR DESCRIPTION
**Pull request by redreceipt**
Adds prettier.

### CODE REVIEW

**Comment by richarddubay** [comment]
@redreceipt Do you want to go ahead and run `prettier` on all the files while you're at it?

**Comment by redreceipt** [comment]
not really. too many directories, I be sure to miss one anyway.

**Comment by IsaacHardy** [comment]
@redreceipt We won't merged this PR until all of the files have been "prettified" so lets punt this issue for now.

**Comment by redreceipt** [comment]
@IsaacHardy every file has now been prettified and tested

File: src/expression-engine/models/ee/model.js [hunk 1]
```diff
@@ -17,12 +17,16 @@ export class EE extends Heighliner {
   contentImages(markup) {
     if (!markup) return [];
 
-    let images = markup.match(/src=".*\.(jpg|jpeg|png)"/gmi);
+    const images = markup.match(/src=".*\.(jpg|jpeg|png)"/gim);
```
Comment (richarddubay) [inline src/expression-engine/models/ee/model.js:20]: @redreceipt Any idea about this change here? It's in a couple of other places too. Mostly concerned with the `/gmi` and `/gim` change. Do you know what this is?

Comment (redreceipt) [reply src/expression-engine/models/ee/model.js:20]: that's a regex match function, those are flags that the order doesn't matter, guessing prettier just sorts them

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Advanced_searching_with_flags_2

Comment (richarddubay) [reply src/expression-engine/models/ee/model.js:20]: Haha! They alphabetized them. 😂

**Review by richarddubay — APPROVED** [review]

**Pull request merged**